File: hubsmsa-backend/hubsmsa-persistence/src/main/java/br/gov/pbh/prodabel/hubsmsa/enumeration/FormaCadastroEnum.java
package br.gov.pbh.prodabel.hubsmsa.enumeration;

public enum FormaCadastroEnum {

	I("Carga Inicial"), 
	C("Cadastro Manual"), 
	A("Carga Inicial Alterada");
	
	private String name;

	private FormaCadastroEnum(String name) {
		this.name = name;
	}

	public String getName() {
		return name;
	}
	
}

File: hubsmsa-frontend/src/app/shared/shared-models/enum/forma-cadastro-enum.ts
export enum FormaCadastroEnum {
  INICIAL = 'I',
  MANUAL = 'C',
  ALTERADA = 'A'
}

export const formaCadastroEnumMensagem = {
  [FormaCadastroEnum.INICIAL]: 'Carga Inicial',
  [FormaCadastroEnum.MANUAL]: 'Cadastro Manual',
  [FormaCadastroEnum.ALTERADA]: 'Carga Inicial Alterada',
};
